src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -224,6 +232,14 @@ struct UpwindFlux {
     using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
     static std::string name() noexcept { return "NormalTimesFluxPi"; }
   };
+  struct NormalTimesGamma2Psi {
+    using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
+    static std::string name() noexcept { return "NormalTimesGamma2TimesPsi"; }
```

Comment:
is it needed to have the different name? if not, you can just omit the name altogether and all of the needed utilities will infer the name from the tag.
(same for below tag)

Reply:
Good point, I shortened their names. But I don't think the names can be omitted entirely as these structs do not inherit `db::SimpleTag`, and that is probably required for auto-name-generation